Docs admission control changes (#8498)
solo-io/gloo: docs/content/guides/traffic_management/configuration_validation/admission_control.md

## Monitor the validation status of Gloo resources

When Gloo Edge fails to process a resource, the error is reflected in the resource's {{< protobuf name="core.solo.io.Status" display="Status">}}. You can run `glooctl check` to easily view any configuration errors on resources that have been admitted to your cluster.

Additionally, you can configure Gloo Edge to publish metrics that record the configuration status of the resources.

In the `observabilityOptions` of the Settings CRD, you can enable status metrics by specifying the resource type and any labels to apply
to the metric. The following example adds metrics for virtual services and upstreams, which both have labels that include the namespace and name of each individual resource:

```yaml
observabilityOptions:
  configStatusMetricLabels:
    Upstream.v1.gloo.solo.io:
      labelToPath:
        name: '{.metadata.name}'
        namespace: '{.metadata.namespace}'
    VirtualService.v1.gateway.solo.io:
      labelToPath:
        name: '{.metadata.name}'
        namespace: '{.metadata.namespace}'
```

After you complete the [Hello World guide]({{% versioned_link_path fromRoot="/guides/traffic_management/hello_world/" %}}) 

> to generate some resources, you can see the metrics that you defined at <http://localhost:9091/metrics>. If the port

Reviewer: ```suggestion
to generate some resources, you can see the metrics that you defined at `[http://localhost:9091/metrics](http://localhost:9091/metrics)`. If the port
```

forwarding is directed towards the Gloo pod, the `default-petstore-8080` upstream reports a healthy state:
```
validation_gateway_solo_io_upstream_config_status{name="default-petstore-8080",namespace="gloo-system"} 0
```

> If the port forwarding is switched to the gateway pod, you can see the metrics defined for virtual services by

Reviewer 2: In gloo edge > 1.12, there is no gateway pod, it has been merged into the gloo pod

> revisiting the metrics endpoint: <http://localhost:9091/metrics>.

Reviewer: ```suggestion
revisiting the metrics endpoint: `[http://localhost:9091/metrics](http://localhost:9091/metrics)`.
```

```
validation_gateway_solo_io_virtual_service_config_status{name="default",namespace="gloo-system"} 0
```

## Disable resource validation in Gloo Edge

> Because the validation admission webhook is set up automatically in Gloo Edge, a `ValidationWebhookConfiguration` resource is created in your cluster. To disable the webhook and prevent the `ValidationWebhookConfiguration` resource from being created, set the following values in your Helm values file:

Reviewer: Maybe add something to explain what this would mean?

```suggestion
Because the validation admission webhook is set up automatically in Gloo Edge, a `ValidationWebhookConfiguration` resource is created in your cluster. You can disable the webhook, which prevents the `ValidationWebhookConfiguration` resource from being created. When validation is disabled, any Gloo resources that you create in your cluster are translated to Envoy proxy config, even if the config has errors or warnings. 

To disable validation, use the following `--set` options during installation, or configure your Helm values file accordingly.
```
